app/graphql/types/query_type.rb

```diff
@@ -64,7 +72,34 @@ def user(id:)
 
     def users
       ability.authorize!(:index, User)
-      User.accessible_by(ability).order(:last_name)
+      User.accessible_by(ability).order(:sort_name)
+    end
+
+    def project_permission_actions
```

Comment:
Sure, I can make that change.  To confirm though, are you imagining that the GraphQL projectPermissionActions query (likely renamed) would then only return the readObjectsAction and manageAction string values?  Sine the concept of those two special permissions exists primarily on the backend, I need to somehow still let the frontend know what the string keys are since I'm dynamically building the permissions UI based on backend-driven data.

Reply:
Marking as resolved because of our separate conversation, and just pushed some updates to the PR.